### PR TITLE
Translations for mailing tile

### PR DESCRIPTION
**Pull request by wipascal**
- Translations for mailing tile
- using new API for value and label of a object field
- set an ID for the maling tile so that monkey patching easier

### CODE REVIEW

File: basxconnect/core/layouts/editlegalperson.py [hunk 1]
```diff
@@ -49,6 +49,7 @@ def mailings_tab():
                     modal_view=LegalPersonEditMailingsView
                 ),
                 editperson.tiling_col(),
-            )
+            ),
+            id="mailing-tab-content",
```
Comment (saemideluxe) [inline basxconnect/core/layouts/editlegalperson.py:53]: Do the ids have a certain use (or future use-case)?

**Review by saemideluxe — APPROVED** [review]

File: basxconnect/core/layouts/editlegalperson.py (hunk 1 above)
Comment (wipascal) [reply basxconnect/core/layouts/editlegalperson.py:53]: Yes, I am using it for patching in the urbanmosaic project. I find this slightly better than just assuming a certain structure.
Do you think it would help to add a comment?

Comment (saemideluxe) [reply basxconnect/core/layouts/editlegalperson.py:53]: I see. Well, I am sure that relying on structure is not the best solution for customization (but it works in all circumstances). However, I am not sure whether relying on IDs is the best approach to handle customizations as well... Issues with IDs are:

- While IDs are less likely to change (in comparison to structure), they still have a chance of beeing changed, e.g. during a refactoring
- IDs needs to be guaranteed unique on the whole page
- It is not clear in which places we should add IDs or not, and on which level of detail.

One alternative would be to use BaseElement.filter and check for the type of the element in order to get the correct one. The filtering could also be improved to allow a more jQuery-style query.
Another approach is to automatically add classes to all HTMLElements, like ```_class="bread-tabs"``` or similar.

But I think we need to explore customizations and how we use them continiously for a few projects. Also, as a note, if we add a standardized concept for customizations to hook in, it would likely be good to implement it directly in bread.

So in short, I think a comment is enough here but let us also keep thinking about more generic approaches for layout customizations.
